scripts/tools/zap_regen_all.py

```diff
@@ -224,12 +224,17 @@ def getTargets(type, test_target):
     return targets
 
 
+def runBootstrap():
+    subprocess.check_call(os.path.join(CHIP_ROOT_DIR, "scripts/tools/zap/zap_bootstrap.sh"), shell=True)
+
+
 def main():
     logging.basicConfig(
         level=logging.INFO,
         format='%(asctime)s %(name)s %(levelname)-7s %(message)s'
     )
     checkPythonVersion()
+    runBootstrap()
```

Comment:
Shouldn't this get skipped if `args.dry_run`?

Reply:
Yes, I think so. I guess the dry run should only print targets which are supposed to be regenerated. Will fix that.